pkgs/desktops/lumina/lumina/default.nix

```diff
@@ -53,16 +70,26 @@ mkDerivation rec {
   dontDropIconThemeCache = true;
 
   patches = [
-    ./LuminaOS-NixOS.cpp.patch
+    (fetchpatch {
+      name = "Add-LuminaOS-NixOS.cpp.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/lumina-desktop/lumina/pull/802.patch";
+      sha256 = "sha256-YgTKEvlWd3Ys9IUxcXqvrfV258mXTW/MRhWLj8pP23o=";
+    })
   ];
 
-  prePatch = ''
-    # Copy Gentoo setup as NixOS setup and then patch it
-    # TODO: write a complete NixOS setup?
-    cp -a src-qt5/core/libLumina/LuminaOS-Gentoo.cpp src-qt5/core/libLumina/LuminaOS-NixOS.cpp
-  '';
-
   postPatch = ''
+    # Fix file paths in NixOS distribution specific source code
+    acpi="${acpi}/bin/acpi" \
+      amixer="${alsa-utils}/bin/amixer" \
```

Comment:
I think it would be easier if we used substituteAll in patchPhase with inherits.